libsolutil/CDCL.cpp

```diff
@@ -230,9 +254,56 @@ std::pair<Clause, size_t> CDCL::analyze(Clause _conflictClause)
 	return {move(learntClause), backtrackLevel};
 }
 
-void CDCL::addClause(Clause _clause)
+void CDCL::addClause(const vector<Literal>& _lits)
 {
-	m_clauses.push_back(make_unique<Clause>(move(_clause)));
+	if (!ok) {
+		return;
+	}
```

Comment:
```suggestion
	if (!ok)
		return;
```
(no braces for a single statement)

Reply:
Thanks, this has now been fixed, very helpful! :)